src/main/java/hanziToAnki/ArgParser.java

```diff
@@ -1,24 +1,21 @@
 package hanziToAnki;
 
-import org.apache.commons.io.FilenameUtils;
-
 import java.util.ArrayList;
 import java.util.List;
 
 public class ArgParser {
 
-    public record ParsedArgs(ExportOptions options, List<String>fileNames, String outputFileName) {}
+    public record ParsedArgs(ExportOptions options, List<String> fileNames, String outputFileName) {}
 
     public static ParsedArgs parseArgs(String[] args) {
         List<String> fileNames = new ArrayList<>();
-        fileNames.add(args[args.length - 1]);
-        String outputFileName = FilenameUtils.removeExtension(fileNames.get(0)) + ".csv";
+        String outputFileName = "output.csv"; //this or stdout?
```

Comment:
how about - default to output with automatic .tsv name, have a flag for stdin in which case we also provide an stdout